Added claymore as a weapon

TaralGit opened the pull request:
Added the claymore as my last contribution for melee weapons. Also, my last adjustments for the combat knife, laser gun and machete sprites.

![Screenshot 2021-06-01 043026claymore](https://user-images.githubusercontent.com/76408146/120332464-9fb79b00-c2a3-11eb-8240-cdd943a28825.png)
Claymore sprite and inhands

Update:

![Screenshot 2021-06-04 193734claymore](https://user-images.githubusercontent.com/76408146/120880206-aa14b600-c57d-11eb-8038-ea15801345be.png)
updated claymore sprite

Update2:
![Screenshot 2021-06-08 034605lastsprites](https://user-images.githubusercontent.com/76408146/121187836-545e3900-c81d-11eb-808f-1e3747815062.png)
All the sprites featured

:cl:
- add: claymore as a melee weapon
- fix: fixed the combat knife, laser gun and machete sprites

mirrorcult commented on Resources/Textures/Objects/Weapons/Melee/claymore.rsi/meta.json:
```diff
@@ -0,0 +1,22 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation and modified by Taral at https://github.com/tgstation/tgstation/pull/31615/commits/1ffa93a49df1e728a4f55d6b867879d29a8bec1a",
```
link didn't work for me, should it be https://github.com/tgstation/tgstation/pull/31615/commits/8795484578b33a9919b7801667287c0f28afe642?

mirrorcult replied:
or https://github.com/tgstation/tgstation/pull/31400/commits/6046fb493bf487a679a235acfebfe3674cf2db8a?

mirrorcult replied:
everything else is good btw

TaralGit replied:
ok. I fixed the commit from the meta.json

SweptWasTaken commented:
Why do you have a commit called machete? Also i'm going to be honest I don't like how the claymore looks. Looks like some kind of light blade instead of metal

TaralGit commented:
> Why do you have a commit called machete? Also i'm going to be honest I don't like how the claymore looks. Looks like some kind of light blade instead of metal

that's for the machete sprite. I will fix the claymore tomorrow by giving the blade a more metalic look.

TaralGit commented:
ok. it's all done

SweptWasTaken commented:
> 
> 
> ok. it's all done

Can you show your changes?

TaralGit commented:
> > ok. it's all done
> 
> Can you show your changes?

Ok. I updated the OP with another screenshot with all the sprites that we're worked on.

SweptWasTaken commented:
The claymore looks great now.

SweptWasTaken reviewed (approved):
?

You removed these laser gun and machete sprites but added the combat knife sprite? If you're going other stuff like this in a PR you need to write it in your description.

TaralGit commented:
> ?
> 
> You removed these laser gun and machete sprites but added the combat knife sprite? If you're going other stuff like this in a PR you need to write it in your description.

Yeah, I basically fixed the sprites for the combat knife, machete, laser gun and added the claymore. The end result is in the last bottom screenshot and I also added all the changes in the changelog, but in future pulls I will describe my commits a little better than just "last touches" or "." .
